convlog/tests/testdata/confusing_nakis_7.json

```diff
@@ -0,0 +1 @@
+{"log":[[[4,0,0],[30200,29600,23600,16600],[24],[34],[13,13,19,27,28,32,35,35,38,39,43,44,47],[47,44,22,"c292728",12,26,21],[44,60,43,22,32,19,60],[12,13,17,19,21,22,23,23,25,32,33,34,36],[31,24,18,24,11,29],[19,17,60,36,"r34",60],[14,16,25,26,27,28,29,31,36,38,39,43,46],[34,15,46,19,13],[43,46,60,60,31],[11,17,17,21,52,25,29,31,35,36,38,39,45],[46,"17p1717",26,32,11,26],[11,21,29,46,45,11],["和了",[-2600,3600,0,0],[1,0,1,"40符2飜2600点","Riichi(1飜)","Dora(1飜)","Ura Dora(0飜)"]]]],"rule":{"disp":"Gold Room South","aka53":1,"aka52":1,"aka51":1},"name":["A","B","C","target"],"title":["",""]}
```

Comment:
You need to modify convlog/tests/testdata/mod.rs to actually include the file.

Reply:
Added.
However, this naki won't make the program exit with error, it just doesn't show one step in the result. Adding it to convlog may not actually help us add regression test for this situation 😂